examples/src/main/java/com/jagrosh/jdautilities/examples/command/AboutCommand.java

```diff
@@ -15,107 +15,205 @@
+    {
+        //Super variables
+        this.name = "About";
+        this.aliases = new String[] {"whoareyou"};
         this.help = "shows info about the bot";
         this.guildOnly = false;
-        this.perms = perms;
         this.botPermissions = new Permission[]{Permission.MESSAGE_EMBED_LINKS};
+
+        //About Command variables
+        this.color = color;
+        this.description = description;
+        this.commands = commands;
     }
-    
-    public void setIsAuthor(boolean value)
+
+
+    @Override
+    protected void execute(CommandEvent event)
     {
-        this.IS_AUTHOR = value;
+        //Get info about the bot
+        event.getJDA().asBot().getApplicationInfo().queue(info -> {
+            if (inviteLink == null) //Set the invite link if not set yet
+                inviteLink = info.isBotPublic() ? info.getInviteUrl(0L, perms) : "";
+            send(event); // Send the About embed
+        }, failure -> {
+            //Log the failure
+            Logger log = LoggerFactory.getLogger("OAuth2");
+            log.error("Could not generate invite link ", failure);
+            inviteLink = "";
+            send(event); //Send the About embed anyway
+        });
     }
-    
-    public void setReplacementCharacter(String value)
+
+    private void send(CommandEvent event)
     {
-        this.REPLACEMENT_ICON = value;
-    }
-    
-    @Override
-    protected void execute(CommandEvent event) {
-        if (oauthLink == null) {
-            try {
-                ApplicationInfo info = event.getJDA().asBot().getApplicationInfo().complete();
-                oauthLink = info.isBotPublic() ? info.getInviteUrl(0L, perms) : "";
-            } catch (Exception e) {
-                Logger log = LoggerFactory.getLogger("OAuth2");
-                log.error("Could not generate invite link ", e);
-                oauthLink = "";
-            }
+        //Save these here so code is less messy
+        JDA jda = event.getJDA();
+        CommandClient cmdClient = event.getClient();
+        User self = event.getSelfUser();
+        //This gets the Bot Owner as a user Mention
+        // Make sure to set this in your Command Client with .setOwnerId()
+        String ownerMention = "<@" + cmdClient.getOwnerId() + ">";
+        String serverInv = cmdClient.getServerInvite();
+
+        StringBuilder sb = new StringBuilder();
+
+        //Setup the basic parts of the EmbedBuilder
+        EmbedBuilder builder = new EmbedBuilder()
+            .setColor(color)
+            .setAuthor(self.getName(), null, self.getAvatarUrl())
+            .setTitle("About " + self.getName() + "!")
+            .setFooter("Last restart", self.getAvatarUrl())
+            .setTimestamp(cmdClient.getStartTime());
+
+        //Use a StringBuilder cuz nice
+        sb.append("Hello! I am **").append(self.getName()).append("**\n")
+          .append("I was made by **").append(ownerMention)
```

Comment:
I'm not sure why you removed the ability to change whether or not the owner is the author; this is used in self-hostable bots where the owner and the author are likely different people,